src/data_info/services/metadata.clj

```diff
@@ -70,8 +70,9 @@
       type)))
 
 (defn- get-readable-data-item
+  "Gets a data-item's path and type and checks that it's readable"
   [cm user data-id]
-  (let [{:keys [path] :as data-item} (uuids/path-stat-for-uuid cm user data-id)]
+  (let [{:keys [path] :as data-item} (stat/uuid-stat cm user data-id :filter-include [:path :type])]
```

Comment:
this filter is chosen by looking at all the callers. One used only the path, most used both path and type.